### PR TITLE
engine/install/ubuntu: update instructions to match debian variant

### PR DESCRIPTION
**Pull request by thaJeztah**
Incorporate the changes made in the debian variant from;

- 6d8b08171c734ff66a277183c6b8fd4a72fb1fab (https://github.com/docker/docs/pull/16654)
- 37ae6721930ee3d96a6235b2d94920518ce14383 (https://github.com/docker/docs/pull/15902)

Signed-off-by: Sebastiaan van Stijn <github@gone.nl>

### CODE REVIEW

File: engine/install/ubuntu.md [hunk 1]
```diff
@@ -232,6 +232,7 @@ download a new file each time you want to upgrade Docker Engine.
    `hello-world` image:
 
    ```console
+   $ sudo service docker start
```
Comment (thaJeztah) [inline engine/install/ubuntu.md:235]: Kept it the same as the `debian` variant, but wondering if we should change this to `sudo systemctl start docker` (as most of our docs assume systemd is used, and `systemctl` is likely the canonical command for that).

Comment (dvdksn) [reply engine/install/ubuntu.md:235]: should be safe to use systemctl, I think. Can't recall receiving any requests to update this, unlike apt/apt-get. When we get around to defining linux/unix/shell conventions, we could consider including a preference to either command there

**Review by dvdksn — APPROVED** [review]

File: engine/install/ubuntu.md (hunk 1 above)
Comment (thaJeztah) [reply engine/install/ubuntu.md:235]: Yup; let's stick to this for now to keep them the same.

I saw some other things we could update (e.g. the examples for versions listed by `apt-cache madison`, and some other bits and bops).

**Comment by thaJeztah** [comment]
Let me bring this one in 👍

**Pull request merged**